bindings/python/py_src/tokenizers/pre_tokenizers/__init__.pyi

```diff
@@ -184,6 +184,40 @@ class Sequence(PreTokenizer):
         """
         pass
 
+class Split(PreTokenizer):
```

Comment:
@n1t0 - what is the best way to re-generate this code? 

At first I ran `make style` which ran `python stub.py` which generated this code. Afterward I made changes to bindings/python/src/pre_tokenizers.rs and ran `make style` again - BUT this code wasn't changed accordingly...I had to manually delete the code and then after re-running `make style` it worked again. Not sure if you have encountered a similar problem before.

Reply:
That's weird. `python stub.py` needs to have the version of `tokenizers` in the env up-to-date, so if you compiled everything before running it, it should have worked fine.